src/components/Rocket.js

```diff
@@ -1,17 +1,21 @@
 /* eslint-disable react/prop-types */
```

Comment:
Please do well to avoid using eslint disable request in your code. So the code reviewers don't give us issues later :smile: